### DISCUSSION TITLE
Reddit provider only returns username even with additional scopes

### DISCUSSION THREAD

**Discussion by ghost** [nextauthjs/next-auth#2571] [Help]
### Description 🐜

Hello. I am following the recommended Reddit boilerplate:

```js
import NextAuth from 'next-auth'
import Provider from 'next-auth/providers'

const options = {
  providers: [
    Provider.Reddit({
      clientId: process.env.REDDIT_CLIENT_ID,
      clientSecret: process.env.REDDIT_CLIENT_SECRET,
      scope: 'identity mysubreddits read',
    }),
  ],
  database: process.env.MONGO_URI,
}

export default (req, res) => NextAuth(req, res, options)
```

```js
import { signIn, signOut, useSession } from 'next-auth/client'

export default function Page() {
  // eslint-disable-next-line no-unused-vars
  const [session, loading] = useSession()

  return (
    <>
      {!session && (
        <>
          Not signed in <br />
          <button onClick={() => signIn()} type='button'>
            Sign in
          </button>
        </>
      )}
      {session && (
        <>
          {console.log(JSON.stringify(session.user))}
          Signed in as {session.user.name} <br />
          <button onClick={() => signOut()} type='button'>
            Sign out
          </button>
        </>
      )}
    </>
  )
}
```

According to the [docs here](https://next-auth.js.org/providers/reddit) Reddit is pre-configured to return id, name and email. Additionally, in my Provider config, I am selecting identity the scopes: identity mysubreddits read. However, logging the session.user shows only the name `{"name":"reddituser892"}`

### Is this a bug in your own project?

No

### How to reproduce ☕️

Configure the example repository to use Reddit as a provider, then log the session.user to see what the object contains after successful login.

### Environment 🖥

  System:
    OS: Windows 10 10.0.19042
    CPU: (64) x64 AMD Ryzen Threadripper 3990X 64-Core Processor 
    Memory: 113.62 GB / 127.87 GB
  Binaries:
    Node: 14.17.3 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD      
    npm: 6.14.13 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1023.0), Chromium (91.0.864.59) 

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
See the profile callback:

https://next-auth.js.org/configuration/providers#options

The doc page you linked also shows you that the default profile callback doesn't set email and image:

https://github.com/nextauthjs/next-auth/blob/main/src/providers/reddit.js
